README.md

```diff
@@ -164,6 +164,7 @@ Options are specified with the -D flag.  The allowed options include:
     ARMADILLO_LIBRARY=(/path/to/armadillo/libarmadillo.so): Armadillo library
     BUILD_CLI_EXECUTABLES=(ON/OFF): whether or not to build command-line programs
     BUILD_PYTHON_BINDINGS=(ON/OFF): whether or not to build Python bindings
+    PYTHON_EXECUTABLE:FILEPATH=(/path/to/python_version): path to specific python executable
```

Comment:
We should update: https://github.com/mlpack/mlpack/blob/a6040b79e17292d11a6436363ba3fdede88e6418/doc/guide/build.hpp#L137 as well.

Reply:
Also, I would say, since none of the other options specify the CMake variable type, it's not necessary to include the `:FILEPATH` here (alternately, we could include the type of every variable; up to you, I think either is fine). :+1: